src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/Messager.java

```diff
@@ -389,16 +432,21 @@ public boolean hasWarnings() {
     }
 
     /**
-     * Prints the error and warning counts, if any.
+     * Prints the error and warning counts, if any, to. the diagnostic writer
```

Comment:
Move period from after "to" to the end of the sentence.

Reply:
Wow, how did that typo come about? :-)